pmp-reactjs-ui/src/nav/SideNav.js

```diff
@@ -55,7 +55,7 @@ function SideNav({ open }) {
     };
     return (
         <div className="flex font-inter bg-white h-screen z-30">
-            <div className={`h-full ${open ? "absolute inset-y-16 left-0 w-64" : "absolute inset-y-16 w-20"} 
+            <div className={`h-full ${open ? "absolute inset-y-16 w-64" : "absolute inset-y-16 w-20"} 
```

Comment:
removing left-0 will this have any impact in expand collapse of side menu, scrolling of page?
check in eng as well as ara